test/racelist_web/features/main_page_test.exs

```diff
@@ -0,0 +1,28 @@
+defmodule Racelist.MainPageTest do
+  use ExUnit.Case
+  use Racelist.FeatureCase
+  use Hound.Helpers
+
+  describe "main page" do
+    
+    test "homepage loads successfully", _meta do
+      navigate_to("/")
+      assert page_title() == "Hello Racelist!"
+    end
+  end
+
+  describe "signing in" do
+    test "successfull signing in with Google", _meta do
+      navigate_to("/")
+      Racelist.TestHelpers.sign_in_user()
+
+      _navigation = find_element(:xpath, ~s|//*[@id="nav-mobile"]|)
+
+      assert current_url() == "http://localhost:4001/#"
```

Comment:
Would be good if there's a configuration file that you can both use to run the application and pull those values here for testing.